<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Print::Internal::PrintCorrectionPreferencesWidget</class>
 <widget class="QWidget" name="Print::Internal::PrintCorrectionPreferencesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Print Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0" colspan="3">
    <widget class="QPushButton" name="printTestWithoutCorrection">
     <property name="text">
      <string>Print the test page without the correction</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Check print errors of your printer and correct them.&lt;br/&gt;
First step, print the test page &lt;u&gt;without&lt;/u&gt; the correction and check the marks.&lt;br/&gt;
Second step, inform the corrections.&lt;br/&gt;
Thrid step, print the test page &lt;u&gt;with&lt;/u&gt; the correction and re-check the marks.&lt;br/&gt;</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Correct margins</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="3">
       <widget class="QComboBox" name="horizCombo">
        <item>
         <property name="text">
          <string>To your right (-&gt;)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>To your left (&lt;-)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>mm</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Horizontal shift</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Vertical shift</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>mm</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Printing direction</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Distance between horizontal line and the left border of the page, in millimeters</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="4">
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Your print</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="currentPrintLabel">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="midLineWidth">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="6" column="3">
       <widget class="QComboBox" name="direction">
        <item>
         <property name="text">
          <string>From top down</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>From bottom up</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="verticCombo">
        <item>
         <property name="text">
          <string>To the top of the page</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>To the bottom of the page</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="horiz"/>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="vertic"/>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Distance between vertical line and the top of the page, in millimeters</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QDoubleSpinBox" name="horizDist"/>
      </item>
      <item row="4" column="3">
       <widget class="QDoubleSpinBox" name="verticDist"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QPushButton" name="printTestWithCorrection">
     <property name="text">
      <string>Print the test page with the correction</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
